service-node/src/test/scala/org/enterprisedlt/fabric/service/node/UtilTest.scala

```diff
@@ -0,0 +1,103 @@
+package org.enterprisedlt.fabric.service.node
+
+import org.enterprisedlt.fabric.service.node.endorsement._
+import org.enterprisedlt.fabric.service.node.model._
+import org.scalatest._
+import Matchers._
+
+
+/**
+ * @author Andrew Pudovikov
+ */
+class UtilTest extends FunSuite {
```

Comment:
please move to EndorsementPolicyCompiler (it would be great to split to compiler and parser tests)